# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/rustc/hotfix-stage0.patch.cross
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- rustc-1.80.0-src/src/bootstrap/src/core/sanity.rs.vanilla	2024-08-02 17:16:05.185021595 +0200
+++ rustc-1.80.0-src/src/bootstrap/src/core/sanity.rs	2024-08-02 17:16:54.409021749 +0200
@@ -271,13 +271,14 @@
                     }
                 }
             }
-
+/*
             if !has_target {
                 panic!(
                     "No such target exists in the target list,
                 specify a correct location of the JSON specification file for custom targets!"
                 );
             }
+*/
         }
 
         if !build.config.dry_run() {
